enet.pyx

```diff
@@ -925,15 +928,31 @@ cdef class Host:
         the host and its peers. The timeout is in milliseconds.
         """
 
+        cdef int number_result
```

Comment:
Why is this necessary? And if it is, could it simply be renamed to 'result' and used in both cases?